src/zenml/models/v2/core/schedule.py

```diff
@@ -46,15 +46,36 @@ class ScheduleRequest(WorkspaceScopedRequest):
     active: bool
 
     cron_expression: Optional[str] = None
-    start_time: Optional[datetime.datetime] = None
-    end_time: Optional[datetime.datetime] = None
-    interval_second: Optional[datetime.timedelta] = None
+    start_time: Optional[datetime] = None
+    end_time: Optional[datetime] = None
+    interval_second: Optional[timedelta] = None
     catchup: bool = False
-    run_once_start_time: Optional[datetime.datetime] = None
+    run_once_start_time: Optional[datetime] = None
 
     orchestrator_id: Optional[UUID]
     pipeline_id: Optional[UUID]
 
+    @field_validator(
+        "start_time", "end_time", "run_once_start_time", mode="after"
+    )
+    @classmethod
+    def _ensure_tzunaware_utc(
```

Comment:
I know we are not handling the update logic of schedules fully, but ideally, the same validator should be applied to the update model too right?

Reply:
I have another branch (workspace-related) that removes all these options from the schedule update model. This should have never existed in the first place, schedules are just read-only currently in ZenML until we implement native schedules.

Reply:
Alright, feel free to resolve then.